Gemfile

```diff
@@ -7,5 +7,4 @@ group :test do
 end
 
 group :profile do
-  gem 'ruby-prof'
```

Comment:
Bundler couldn't complete a bundle install with it. It couldn't compile the native code it needed because first jruby needed the right parameters passed to it and if those were correct then it couldn't find the headers it needed.